mail/api/mail.py

```diff
@@ -482,113 +271,83 @@ def get_user_addresses(user: str | None = None) -> list:
 
 
 @frappe.whitelist()
-def get_mime_message(mail_type: MailType, name: str) -> dict:
+def get_mime_message(name: str) -> dict:
 	"""Fetches mail mime message and related data."""
 
-	doc = frappe.get_doc(mail_type, name)
+	doc = frappe.get_doc("Email Message", name)
 
 	def get_mail_recipients(recipient_type):
 		return ", ".join([d.email for d in doc.recipients if d.type == recipient_type])
 
-	mail = {
-		"message": doc.message,
+	pass_or_fail = {1: _("'Pass'"), 0: _("'Fail'")}
+
+	return {
+		"message": doc.get_mime_message(),
```

Comment:
```suggestion
		"message": doc.message or doc.get_mime_message(),
```

`doc.get_mime_message()` always fetches and returns.